src/sql/workbench/contrib/dashboard/browser/pages/databaseDashboardPage.contribution.ts

```diff
@@ -96,7 +98,8 @@ export const databaseDashboardSettingSchema: IJSONSchema = {
 			},
 			widget: {
 				'tasks-widget': [
-					'newQuery',
+					NewQueryTask.ID,
```

Comment:
Updated this line to use the actual ID from the query task so that if we ever update the ID it will get reflected here.